x/state-verifier/types/keys.go

```diff
@@ -0,0 +1,25 @@
+package types
+
+import "strconv"
+
+const (
+	// ModuleName defines the module name
+	ModuleName = "state-verifier"
+
+	// StoreKey defines the primary module store key
+	StoreKey = ModuleName
+)
+
+const (
+	prefixConsensusStateKey = iota + 1
+)
+
+var ConsensusStateKey = []byte{prefixConsensusStateKey}
+
+func GetConsensusStateKey(height int64) []byte {
+	return append(ConsensusStateKey, []byte(strconv.FormatInt(height, 10))...)
+}
+
+func ExtractHeightFromConsensusStateKey(key []byte) (int64, error) {
```

Comment:
unused? if needed would be good to have few tests on that

Reply:
removed